compute_tools/src/http/api.rs

```diff
@@ -133,6 +137,38 @@ async fn routes(req: Request<Body>, compute: &Arc<ComputeNode>) -> Response<Body
             }
         }
 
+        (&Method::GET, "/schema/objects") => {
+            info!("serving /schema/objects GET request",);
+            match get_schema_objects(compute).await {
+                Ok(res) => render_json(Body::from(serde_json::to_string(&res).unwrap())),
+                Err(_) => render_json_error(
+                    "can't get schema objects",
+                    StatusCode::INTERNAL_SERVER_ERROR,
+                ),
+            }
+        }
+
+        (&Method::GET, "/schema/dump") => {
+            let db_name = match must_get_query_param(&req, "db_name") {
+                Err(e) => return e.into_response(),
+                Ok(db_name) => db_name,
+            };
+            info!("serving /schema/dump GET request with db_name: {db_name}",);
+            match schema_dump(compute, &db_name).await {
+                Ok(res) => Response::builder()
+                    .header(CONTENT_TYPE, "text/plain")
+                    .body(Body::wrap_stream(res))
+                    .unwrap(),
```

Comment:
Maybe it would be good to add a `render_plain` function similar to the `render_json` function you added?

Reply:
Ok. Btw, we do not return content-type in other API responses, probably we need to fix it, but it's out of the scope of this PR, so I decided not to introduce these changes.

Reply:
Yep, that is definitely something that should be fixed. Can you create a ticket and assign it to me?

Reply:
done https://github.com/neondatabase/neon/issues/7629